### PR TITLE
Implement get opinon intent

### PR DESCRIPTION
**Pull request by oilnam**
@tomrf1 `generatePreamble()` and `followupQuestion()` are almost a copy-paste from the headlines version. If you have something similar for reviews it might be a good idea to factor them out.

### CODE REVIEW

File: src/intentLogic/getOpinion.js [hunk 1]
```diff
@@ -3,29 +3,75 @@ const asJson = require('simple-get-promise').asJson;
...
+    };
+
+    var updatePositionalContent = (json) => {
+        attributes.positionalContent = json.response.results.slice(attributes.moreOffset, attributes.moreOffset+3).map(result =>
+            result.id );
+    };
+};
+
+var buildCapiQuery = (searchTerm) => {
+    return helpers.capiQuery('search', '&show-fields=standfirst,byline,headline&tag=commentisfree/commentisfree', searchTerm);
```
Comment (tomrf1) [inline src/intentLogic/getOpinion.js]: You could specify `&page-size=${helpers.pageSize}&page=attributes.moreOffset`, then you wouldn't have to do `json.response.results.slice(attributes.moreOffset, attributes.moreOffset+3)`

File: test/fixtures/getOpinionOn.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
 			"slots": {
 				"search_term": {
 					"name": "search_term",
-					"value": "brexit"
+					"value": "Brexit"
```
Comment (tomrf1) [inline test/fixtures/getOpinionOn.json]: I know it's only test json, but tab indentation is always morally wrong

File: src/intentLogic/getOpinion.js (hunk 1 above)
Comment (oilnam) [inline src/intentLogic/getOpinion.js]: True but that logic is almost a copy of what we already for headlines. I would be tempted to keep it the same across multiple intents if possible to reduce the mental overhead of keeping track of how we're doing things how, if that makes sense.

**Comment by tomrf1** [comment]
👍

**Pull request merged**